scripts/install.sh

```diff
@@ -251,12 +251,12 @@ install_cuda_driver_yum() {
         dnf)
             DNF_VERSION=$(dnf --version | grep -E -o -m 1 "dnf[4-5]")
             if [ "$DNF_VERSION" = "dnf5" ]; then
-                ADD_REPO_FMT="--addrepo"
+                DNF_COMPATIBILITY_FMT="addrepo --from-repofile="
```

Comment:
I believe the space after the `from-repofile` flag will cause issues, this results in the command format: `... addrepo --from-repofile= https://developer.download.nvidia.com...`

Reply:
I made a new commit, could you confirm there is no more space after the equals please? Thank you!